includes/class-coblocks-form.php

```diff
@@ -588,10 +590,37 @@ public function render_field_checkbox( $atts ) {
 
 		$label      = isset( $atts['label'] ) ? $atts['label'] : __( 'Select', 'coblocks' );
 		$label_slug = $checkbox_count > 1 ? sanitize_title( $label . '-' . $checkbox_count ) : sanitize_title( $label );
+		$required   = ( isset( $atts['required'] ) && $atts['required'] );
+
+		if ( $checkbox_count === 1 && $required ) {
+
+			wp_enqueue_script(
+				'coblocks-checkbox-required',
+				CoBlocks()->asset_source( 'js' ) . 'coblocks-checkbox-required.js',
+				array( 'jquery' ),
```

Comment:
I left another comment regarding keeping jQuery as a dependency here. In the interest of continuing to work toward https://github.com/godaddy-wordpress/coblocks/issues/1365, we should ideally avoid the use of jQuery all together.